lib/client/client.go

```diff
@@ -111,67 +111,12 @@ func (proxy *ProxyClient) GetSites() ([]services.Site, error) {
 	return sites, nil
 }
 
-// GenerateCertsForCluster generates certificates for the user
-// that have a metadata instructing server to route the requests to the cluster
-func (proxy *ProxyClient) GenerateCertsForCluster(ctx context.Context, routeToCluster string) error {
```

Comment:
Note to reviewers: this method was unused and basically a subset of `ReissueUserCerts`